.travis.yml

```
@@ -5,6 +5,16 @@ language: c
 
 matrix:
   include:
+   - os: linux
+     env: CLANG="tidy"
+     script:
+      - mkdir build
+      - cd build
+      - cmake ..
+      - make clang-tidy-all-libsettings
```

I think this should work without having to specify `libsettings` in the target name:
```suggestion
      - make clang-tidy-all
```